src/main/scala/com/advancedtelematic/campaigner/http/GroupUpdateResolver.scala

```diff
@@ -28,7 +28,7 @@ class GroupUpdateResolver(deviceRegistry: DeviceRegistryClient, resolver: Resolv
     }
 
     for {
-      devices <- fetchDevices(offset = 0, limit = 50)
```

Comment:
Can we please have names back? Improves readability IMO

Reply:
Names are back :)